docs/docs-content/integrations/vault.md

```diff
@@ -12,94 +12,115 @@ tags: ['packs', 'vault', 'security']
+You can do so by following these steps:
 
-</TabItem>
+1. Log in to [Palette](https://console.spectrocloud.com).
 
-<TabItem label="0.3.x" value="0.3.x">
+2. Navigate to the left **Main Menu** and select **Clusters**.
 
-* **0.3.1**
+3. Select the cluster that has Vault installed to view its details page.
 
-</TabItem>
-</Tabs>
+4. Download the cluster **kubeconfig** file. 
 
+5. Set up your local kubectl environment to use the **kubeconfig** file you downloaded. Review the [Access Cluster with CLI](../clusters/cluster-management/palette-webctl.md) guide for additional guidance. 
 
-## Components
+6. You need to get the Vault namespace and application name. Issue the following command to get the unique values.
 
-Vault integration has the following components:
+    <br />
 
-* Vault server.
-* UI (Optional).
-* [Agent injector](https://www.vaultproject.io/docs/platform/k8s/injector/) (Optional).
+    ```shell
+    VAULT_NAMESPACE=$(kubectl get pods --selector app.kubernetes.io/name=vault --all-namespaces --output jsonpath='{.items[0].metadata.namespace}') && \
+    APP_NAME=$(echo "$VAULT_NAMESPACE" | sed 's/-ns$//')
+    ```
 
+7. Set up port forwarding by issuing the following command so you can access the Vault UI:
 
+    ```
+    kubectl port-forward $APP_NAME 8200:8200 --namespace $VAULT_NAMESPACE
+    ```
 
-## Supported Use cases
+8. Open your browser and access the [Vault UI](https://localhost:8200/ui. You will receive a warning due to using a self-signed certificate, but you can ignore this warning. Follow the prompts on the UI to initialize your root token.
 
-1. Running a Vault Service:
-    * Vault is set up to run in **Dev mode** by default and so, Vault will be unsealed and initialized.
-    * For production use cases, we recommend disabling Dev mode and enable HA.
-    * Also, see [Production Checklist](https://www.vaultproject.io/docs/platform/k8s/helm/run#architecture) recommendations.
-1. Injecting application secrets from an external Vault into pods (**Agent Injector**).
-    * For running agent injector alone in the cluster, use v0.6.0 of Vault pack.
-    * Make sure to set `injector.externalVaultAddr` to point to the external Vault server.
+:::tip
 
-## How secrets are injected in deployments?
+If you don't want to use the Vault UI, you can also initialize and unseal Vault using the Vault CLI or API. For more information, refer to [Vault documentation](https://developer.hashicorp.com/vault/docs/platform/k8s/helm/run#initialize-and-unseal-vault).  
 
-In Kubernetes clusters with Vault integrated, secrets can be injected into the application pods by adding the following annotations:
+:::
 
-```yaml
-vault.hashicorp.com/agent-inject: "true"
-vault.hashicorp.com/agent-inject-secret-<unique_name>: /path/to/secret
-vault.hashicorp.com/role: "<role using which the secret can be fetced>"
+### Terraform
+
+You can reference the Vault pack in Terraform with a data resource:
+
+```tf
```

Comment:
```suggestion
```hcl
```

Reply:
The Terraform example doesn't display like the others in the docs. Maybe if you use `hcl`? I looked at the kubernetes example for PXK.